Pal/src/host/Linux-SGX/sgx_main.c

```diff
@@ -644,6 +646,8 @@ int load_manifest (int fd, struct config_store ** config_ptr)
 
     if (ret < 0) {
         SGX_DBG(DBG_E, "can't read manifest: %s\n", errstring);
+        INLINE_SYSCALL(munmap, 2, config_raw, nbytes);
+        free(config);
```

Comment:
I think this function is right on the cusp of needing a single error case at the end with gotos.  I think the code is ok as-is, but I think one more system call/cleanup case would tip this into having one exception case at the end.